vivisect/__init__.py

```diff
@@ -1890,11 +1890,32 @@ def getNames(self):
         """
         return self.name_by_va.items()
 
-    def getName(self, va):
+    def getName(self, va, smart=False):
         '''
         Returns the name of the specified virtual address (or None).
+
+        Smart mode digs beyond simple name lookups, as follows:
+        If va falls within a known function in the workspace, we return "funcname+<delta>".
+        If not, and the va falls within a mapped binary, we return "filename+<delta>"
         '''
-        return self.name_by_va.get(va, None)
+        name = self.name_by_va.get(va)
+
```

Comment:
lets go with the convention of returning name immediately if one was returned ( this allows all the code from here down to be one "indent" less deep. )